SPECS/kernel-mos/kernel-mos.spec

```diff
@@ -0,0 +1,388 @@
+%undefine _unique_build_ids
+%undefine _unique_debug_names
+%global _missing_build_ids_terminate_build 1
+%global _no_recompute_build_ids 1
+
+%define arch x86_64
+%define archdir x86
+%define config_source %{SOURCE1}
+Summary:        Linux Kernel for MOS
+Name:           kernel-mos
+Version:        5.15.136.1
+Release:        1%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Kernel
+URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
+Source0:        https://github.com/microsoft/CBL-Mariner-Linux-Kernel/archive/rolling-lts/%{mariner_tag}/%{version}.tar.gz#/kernel-mos-%{version}.tar.gz
+Source1:        config
+Source2:        cbl-mariner-ca-20211013.pem
+Source3:        sha512hmac-openssl.sh
+BuildRequires:  audit-devel
+BuildRequires:  bash
+BuildRequires:  bc
+BuildRequires:  build-essential
+BuildRequires:  diffutils
+BuildRequires:  dwarves
+BuildRequires:  elfutils-libelf-devel
+BuildRequires:  flex
+BuildRequires:  gettext
+BuildRequires:  glib-devel
+BuildRequires:  grub2-rpm-macros
+BuildRequires:  kbd
+BuildRequires:  kmod-devel
+BuildRequires:  libcap-devel
+BuildRequires:  libdnet-devel
+BuildRequires:  libmspack-devel
+BuildRequires:  openssl
+BuildRequires:  openssl-devel
+BuildRequires:  pam-devel
+BuildRequires:  pciutils-devel
+BuildRequires:  procps-ng-devel
+BuildRequires:  python3-devel
+BuildRequires:  sed
+Requires:       filesystem
+Requires:       kmod
+Requires(post): coreutils
+Requires(postun): coreutils
+ExclusiveArch:  x86_64
+# When updating the config files it is important to sanitize them.
+# Steps for updating a config file:
+#  1. Extract the linux sources into a folder
+#  2. Add the current config file to the folder
+#  3. Run `make menuconfig` to edit the file (Manually editing is not recommended)
+#  4. Save the config file
+#  5. Copy the config file back into the kernel spec folder
+#  6. Revert any undesired changes (GCC related changes, etc)
+#  8. Build the kernel package
+#  9. Apply the changes listed in the log file (if any) to the config file
+#  10. Verify the rest of the config file looks ok
+# If there are significant changes to the config file, disable the config check and build the
+# kernel rpm. The final config file is included in /boot in the rpm.
+
+%description
+The kernel package contains the Linux kernel.
+
+%package devel
+Summary:        Kernel Dev
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       gawk
+Requires:       python3
+Obsoletes:      linux-dev
+
+%description devel
+This package contains the Linux kernel dev files
+
+%package drivers-accessibility
+Summary:        Kernel accessibility modules
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+
+%description drivers-accessibility
+This package contains the Linux kernel accessibility support
+
+%package drivers-gpu
```

Comment:
Just checking, do the AMD drivers show up in this `kernel-mos-drivers-gpu` subpackage or the main `kernel-mos` package?

Reply:
Yes! I updated the PR description with a screenshot